### PR TITLE
Controller info endpoint and API integration examples

### PR DESCRIPTION
**Pull request by edigaryev**
Resolves https://github.com/cirruslabs/orchard/issues/43, resolves https://github.com/cirruslabs/orchard/issues/55.

### CODE REVIEW

File: pkg/client/events.go [hunk 1]
```diff
@@ -20,7 +20,7 @@ func NewEventStreamer(client *Client, endpoint string) *EventStreamer {
 	streamer := &EventStreamer{
 		client:        client,
 		endpoint:      endpoint,
-		eventsChannel: make(chan v1.Event),
+		eventsChannel: make(chan v1.Event, 64),
```
Comment (fkorotkov) [inline pkg/client/events.go:23]: what is it for?

File: IntegrationGuide.md [hunk 2]
```diff
@@ -6,6 +6,179 @@ You can run `orchard dev` locally and navigate to http://127.0.0.1:6120/v1/ for
 
 ![](docs/orchard-api-documentation-browser.png)
 
+## Using the API
+
+Since everything is relative, we'll show how to consume the API using Python's request library (lower level abstraction) and our Golang package (higher level abstraction) in the hope that this will aid in understanding.
```
Comment (fkorotkov) [inline IntegrationGuide.md:11]: Maybe rephrase it to something like:

```suggestion
Below you'll find examples of using Orchard API via vanilla Python's request library and Golang package that Orchard CLI build on top of.
```

File: pkg/client/events.go (hunk 1 above)
Comment (edigaryev) [reply pkg/client/events.go:23]: This [enables buffering](https://go.dev/tour/concurrency/3), so that the `readAvailableEvents()` method's behavior that tries to read multiple events from the channel actually makes sense.

Comment (fkorotkov) [reply pkg/client/events.go:23]: Right. For some reason in the morning I though it has buffering already. Thanks for fixing it!

**Review by fkorotkov — APPROVED** [review]
